UnitTests/BraintreePayPalNativeCheckoutTests/BTPayPalNativeCheckoutAccountNonce_Tests.swift

```diff
@@ -0,0 +1,87 @@
+import XCTest
+import BraintreeCore
+@testable import BraintreePayPalNativeCheckout
+
+final class BTPayPalNativeCheckoutAccountNonce_Tests: XCTestCase {
+
+    func testPayPalNativeCheckoutNonce_withAllJSON_createsPayPalNativeCheckoutNonce_withAllValues() {
```

Comment:
Nitpicks -

Since the method we're testing here is the `init()`, the test should start with `testInit_...`

Also, I think we generally try to follow the 2 underscore pattern `test<methodName>_<condition>_<result>`